site/releases/2024-dec-24/release-notes.qmd

```diff
@@ -0,0 +1,413 @@
+
+::: {.w-60-ns}
+We've added the capability to build custom **{{< fa home >}} Dashboard**s to allow quick access to information relevant to you via widgets.
+
+:::
+
+::: {.w-40-ns .tc}
+[Customize your dashboard](/guide/configuration/customize-your-dashboard.qmd){.button .button-green}
+
+:::
+
+::::
+
+Widget types include saved inventory and model findings views[^1], analytics reports,[^2] and model activity.[^3] 
+
+![Custom `High Risk` dashboard showing widgets for Tier 1 models and High Severity findings](custom-dashboard.png){width=80% fig-alt="A screenshot of a custom `High Risk` dashboard showing widgets for Tier 1 models and High Severity findings" .screenshot}
+
+### Documentation 
+
+#### User guide updates
+
+<!---
+PR #544: Add login button to docs site
+URL: https://github.com/validmind/documentation/pull/544
+Labels: documentation
+--->
+
+<!-- https://github.com/validmind/documentation/pull/580 -->
+
+##### Log In button
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+We've added the option to log in to the {{< var validmind.platform >}} directly from our documentation site. Click the **Log In** drop-down menu in the top right-hand corner to try it out! 
+
+:::
+
+::: {.w-30-ns .tc}
+[Log in to {{< var vm.product >}}](/guide/configuration/log-in-to-validmind.qmd){.button .button-green}
+
+:::
+
+::::
+
+![New `Log In` button](log-in.png){width=80% fig-alt="A screenshot of the new `Log In` button" .screenshot}
+
+<!-- MANUAL ADD
+https://github.com/validmind/documentation/pull/558 -->
+
+##### New FAQ
+
+::: {.column-margin}
+![New FAQ](faq.png){fig-alt="A screenshot of the new FAQ" .screenshot}
+:::
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-80-ns}
+We've completely renovated our frequently asked questions (FAQ) with completely up-to-date information and new topics such as:
+
+:::
+
+::: {.w-20-ns .tc}
+[FAQ](/faq/faq.qmd){.button .button-green}
+
+:::
+
+::::
+
+- Access & permissions
+- Validation & findings
+- Collaboration
+- Monitoring & reporting
+
+<!-- MANUAL ADD
+https://github.com/validmind/documentation/pull/582 -->
+
+##### {{< var vm.product >}} Help Center 
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-80-ns}
+We've soft-launched our Help Center, enabling you to easily contact us through your method of choice.
+
+:::
+
+::: {.w-20-ns .tc}
+[Support](/support/support.qmd){.button .button-green}
+
+:::
+
+::::
+
+Whether you submit a ticket through our questionnaire, chat with us through the documentation site, send us an email, or give us a call, your requests will all be captured in our Help Center. 
+
+![{{< var vm.product >}} Help Center](help-center.png){width=80% fig-alt="A screenshot of the ValidMind Help Center" .screenshot}
+
+:::
+
+## Enhancements
+
+### {{< var validmind.platform >}} (v1.27.17)
+
+<!---
+PR #1100: Mnt analytics improvements
+URL: https://github.com/validmind/frontend/pull/1100
+Labels: enhancement
+--->
+#### Additional custom analytics settings
```

Comment:
"Enhanced Reporting & Analytics settings"

Reply:
Cool, I will adjust that!